cmd/mcp/toolbox/get_logs.go

```diff
@@ -21,67 +23,116 @@ const (
 // NewGetKubernetesLogsTool creates a new tool for retrieving the pod logs.
 func (m *Manager) NewGetKubernetesLogsTool() SystemTool {
 	return SystemTool{
-		Tool: mcp.NewTool(ToolGetKubernetesLogs,
-			mcp.WithDescription("This tool retrieves the the most recent logs of a Kubernetes pod."),
-			mcp.WithString("pod_name",
-				mcp.Description("The name of the pod."),
-				mcp.Required(),
-			),
-			mcp.WithString("container_name",
-				mcp.Description("The name of the container."),
-				mcp.Required(),
-			),
-			mcp.WithString("pod_namespace",
-				mcp.Description("The namespace of the pod."),
-				mcp.Required(),
-			),
-			mcp.WithNumber("limit",
-				mcp.Description("The maximum number of log lines to return. Defaults to 100."),
-			),
-		),
+		Tool: &mcp.Tool{
+			Name:        ToolGetKubernetesLogs,
+			Description: "This tool retrieves the the most recent logs of a Kubernetes pod.",
+			InputSchema: map[string]interface{}{
```

Comment:
Could we generate the schema from the args struct? Maybe have an interface for the tool args with functions for generating the schema, validation and defaults.